### ISSUE TITLE
Credentials elasticsearch are ignored in es6 configuration

### ISSUE THREAD

**Issue by shofstee** [Netflix/conductor#2896]
**Describe the bug**
According to the documentatie you can configure the username and password for the elasticsearch configuration. You can do this by setting conductor.elasticsearch.username and conductor.elasticsearch.password. This works if you're using the elasticsearch 7 configuration, but not when using the elasticsearch 6 configuration

**Details**
Conductor version: 3.6.1 (build from the main branch)
When conductor.indexing.enabled is enabled and the username and password are set the application crashes with an unauthorized exception  when starting. 

```
conductor.indexing.enabled=true
conductor.elasticsearch.version=6
conductor.elasticsearch.url=...
conductor.elasticsearch.username=...
conductor.elasticsearch.password=...

```

**To Reproduce**
1. Configure conductor to user elasticsearch with a username and password
2. Start conductor-server 
3. Watch the logs for Unauthorized error

**Expected behavior**

**Screenshots**

```
conductor-server  | Caused by: org.elasticsearch.client.ResponseException: method [GET], host [http://es:9200], URI [/_cluster/health?timeout=30s&wait_for_status=yellow], status line [HTTP/1.1 401 Unauthorized]
conductor-server  | Unauthorized
conductor-server  |     at org.elasticsearch.client.RestClient$1.completed(RestClient.java:552) ~[elasticsearch-rest-client-6.8.12.jar!/:6.8.12]
conductor-server  |     at org.elasticsearch.client.RestClient$1.completed(RestClient.java:537) ~[elasticsearch-rest-client-6.8.12.jar!/:6.8.12]
conductor-server  |     at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:122) ~[httpcore-4.4.14.jar!/:4.4.14]

```

**Additional context**
I fixed it by copying the configuration from com.netflix.conductor.es7.config.ElasticSearchV7Configuration to com.netflix.conductor.es6.config.ElasticSearchV6Configuration concerning username and password. That fixed the problem
[0001-Fix-credentials-for-es6.patch.txt](https://github.com/Netflix/conductor/files/8417664/0001-Fix-credentials-for-es6.patch.txt)

**Comment by manan164** [comment]
Hi @shofstee , Fell free to raise the PR to fix this. Thanks.

**Comment by apanicker-nflx** [comment]
Fixed in #2900

**Comment by apanicker-nflx** [comment]
@shofstee Please add your company name to the users list - #2706
